### ISSUE TITLE
Click-to-set in LFO and Alias doesn't work on LInux

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge-rack#668]
As discussed over in #559 

basically on macos I get a release event in onButton and on linux I dont

If I apply the patch below I get a click to set working on linux but then I lose drag

so I'll have to re-implement a slider i suppose to make sure i get all the events i want. sigh.

```
iff --git a/src/LFO.cpp b/src/LFO.cpp
index 636167b..0050917 100644
--- a/src/LFO.cpp
+++ b/src/LFO.cpp
@@ -318,6 +318,7 @@ struct LFOStepWidget : rack::Widget, style::StyleParticipant
         rack::Vec buttonPos;
         void onButton(const ButtonEvent &e) override
         {
+               std::cout << "onButton " << e.action << " " << e.pos.x << " " << e.pos.y << std::endl;
             if (e.action == GLFW_PRESS)
                 buttonPos = e.pos;
 
@@ -336,11 +337,15 @@ struct LFOStepWidget : rack::Widget, style::StyleParticipant
                         val = (1 - val) * 2 - 1;
                     if (getParamQuantity())
                         getParamQuantity()->setValue(val);
+                   e.consume(this);
                 }
                 buttonPos = rack::Vec(-1, -1);
             }
 
             SliderKnob::onButton(e);
+           if (e.isConsumed() && e.action == GLFW_PRESS)
+                e.unconsume();
+
         }
```

**Comment by baconpaul** [comment]
Oh and tagging @contemporaryinsanity here who found this.

**Comment by baconpaul** [comment]
(I'll also note it might just be easier to write a single widget with 16 bars and then i could support all the drag gestures across bars and stuff; so I'll probably just do that if I'm writing from scratch which would also close #559 but I might not. We will see!)

**Comment by baconpaul** [comment]
gonna close the tactical fix and just do the full thing like in #559
